packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -241,8 +144,12 @@ export const createSeatManager = (
     'zcfSeat',
     proposal => ({ proposal }),
     {
-      getNotifier: ({ self }) =>
-        E(zoeInstanceAdmin).getSeatNotifier(zcfSeatToSeatHandle.get(self)),
+      getSubscriber: context => {
+        const { self } = context;
```

Comment:
Once we switch from kinds to farClasses/farInstances, methods such as this will start out looking more like 

```suggestion
      getSubscriber: () => {
         const { self } = this;
```

which is more like your original rather than my suggested change. Nevertheless, uniformity is the higher immediate priority so I'm glad you did this.